sympy/printing/pycode.py

```diff
@@ -822,6 +822,10 @@ def _print_Mod(self, expr):
         return "%s(%s)" % (self._module_format('numpy.mod'), ', '.join(
             map(lambda arg: self._print(arg), expr.args)))
 
+    def _print_Heaviside(self, expr):
+        heaviside0 = 0 # Value of heaviside at x=0. Options include 0, 1/2, 1
```

Comment:
I don't really see a way around picking a default value here, but I think it should be configurable as a printer setting. I also don't know if 0 is the best default. Both Wikipedia and WolframAlpha use 1 as the default.

Reply:
Sympy's Heaviside function gives the option to specify the value at 0:
```python
In [3]: h = Heaviside(x, 3)                                                                                                       

In [4]: h                                                                                                                         
Out[4]: θ(x, 3)

In [5]: h.subs(x, 0)                                                                                                              
Out[5]: 3
```

Reply:
Ah I didn't realize that. So perhaps the best solution here is to require the second argument to be specified to use lambdify. One can replace `Heaviside(x)` with `Heaviside(x, a)` using `expr.replace(Heaviside, lambda x: Heaviside(x, a))`

Reply:
The reason that we needed to pick a default value is largely because the numpy heaviside function requires the second argument (which behaves like Sympy's optional argument). The reason I favored picking a default value is to simplify the process for an end user (which was me). in my case, I was using inverse Laplace transforms, which often return Heaviside functions; for my purposes (and for most purposes, as mentioned on Wikipedia) the exact value chosen at 0 is basically irrelevant.

So: while I see a purpose in making a printer option (or an option somewhere else) to choose Heaviside(0), I think it unnecessarily introduces an extra step for the majority of users if we require the second argument. I am fine with picking a different default, if 1/2 or 1 seems more standard.

Reply:
We can do something like 

```py
if len(expr.args) == 1:
    raise ValueError("The NumPy lambdify printer requires Heaviside(0) to be defined via the second argument to Heaviside (like 'Heaviside(x, 1/2)').")
return "%s(%s,%s)" % (self._module_format('numpy.heaviside'), self._print(expr.args[0]), self._print(expr.args[1]))
```